c/src/proactor/win_iocp.cpp

```diff
@@ -1514,7 +1514,7 @@ void pni_iocp_initialize(void *obj)
   pni_shared_pool_create(iocp);
   iocp->completion_port = CreateIoCompletionPort(INVALID_HANDLE_VALUE, NULL, 0, 0);
   assert(iocp->completion_port != NULL);
-  iocp->zombie_list = pn_list(PN_OBJECT, 0);
+  iocp->zombie_list = pn_list(PN_CLASSCLASS(pni_iocpdesc), 0);
```

Comment:
I noticed that this one is not like the others, so checking if it's as it should be.  The others pass the address of PN_CLASSCLASS.

Reply:
Good spot - that's probably my windows build failure right there - thanks!